app/assets/javascripts/jquery.plug-google-content.js

```diff
@@ -38,7 +38,7 @@
         $.ajax({
           type: 'GET',
           url: batchBooksApiUrl,
-          async: false,
+          async: true,
```

Comment:
We can see what happens if we remove it (assuming it doesn't have unintended affects).